client/tests/e2e/specs/client_area_custom_order_statistics.spec.ts

```diff
@@ -71,59 +57,11 @@ msTest('Test initial status for all orgs', async ({ clientAreaCustomOrder }) =>
   await expect(orgSelector).toHaveText('BlackMesa-2');
 });
 
-msTest('Custom order stats generic error', async ({ clientAreaCustomOrder }) => {
```

Comment:
Why are all these tests removed? I get that the statistics page is not supposed to be accessed for an organization that's not bootstrapped but
1. it can still be the case (change the URL, switch between a bootstrapped org and a not-bootstrapped org, ...)
2. a bootstrapped org doesn't guarantee that the network requests will not fail.